cmd/policy-assistant/pkg/matcher/traffic.go

```diff
@@ -58,29 +58,32 @@ func labelsToString(labels map[string]string) string {
 	return strings.Join(slice.Map(format, slice.Sort(maps.Keys(labels))), "\n")
 }
 
-func (t *Traffic) PrettyString() string {
-	if t == nil || t.Source == nil || t.Destination == nil {
-		return "<undefined>"
+// Helper function to generate the string for source or destination
+func (t *Traffic) formatPeer(peer *TrafficPeer) string {
```

Comment:
nit:
```suggestion
func formatPeer(peer *TrafficPeer) string {
```

Reply:
Can't do this: 
![image](https://github.com/user-attachments/assets/cede3fa4-209e-42da-943d-1bbd1e49b3f0)

Reply:
I think you could remove the `t.` but this is just a style nit and not a blocker